content/en/docs/Advanced/sys-extensions.md

```diff
@@ -62,8 +66,80 @@ This will generate a signed+verity sysextension that can then be used by sysext
 Some extension examples are available under https://github.com/Itxaka/sysext-examples for k3s and sbctl.
 
 
-### Verifying the system extensions
+### Building system extensions from a docker image with enki
+
+{{% alert title="Warning" color="warning" %}}
+This feature is in preview state and only available in Enki from version 0.1.4
+{{% /alert %}}
+
+
+You can also build a system extension from a docker image directly by using [enki](https://github.com/kairos-io/enki) and using a dockerfile to isolate the artifacts you want converted into a system extension.
+
+Notice that when converting a docker image into a system extension, the last layer is the only one converted (The last command in a given Dockerfile) so have that in mind. This is useful for packages that ONLY install things in /usr or manual installation under /usr
+
+
+For example for a given Dockerfiles as such:
+
+```dockerfile
+FROM anchore/grype:latest AS grype
+
+
+FROM scratch
+COPY --from=grype /grype /usr/local/bin/grype
+```
 
+Only the files added in the last step will be converted to a sysext, so the contents of the sysext would be the `/usr/local/bin/grype` binary only.
```

Comment:
Not really. The file with the content `id=_ANY` is auto-generated. We'd better mention that here because while reviewing the PR I didn't know that and it makes is a real pain to create that file too with heredoc in the dockerfile and still keep everything in one command (last layer).

Reply:
Good point. I did not add it as it seems superfluous to me, but the point that you created the file yourself without knowing that the command would create it automatically says that this info is probably needed here. Will add it somewhere around.